fix: Increase MobileBottomBar height to not overlap with iPhone bottom bar

Jdecristi opened the pull request:
## General Notes
Fixes: #6557 

Increased MobileBottomBar height to 70px

## Screen Shots
| Before  |  After PWA |  After Browser |
|:-------:|:-----------:|:--------------:|
| <img width="400" alt="Screenshot 2023-05-12 at 3 53 56 PM" src="https://github.com/Uniswap/interface/assets/89173284/8dec1494-5328-417f-9e9d-1b448a3fc59b"> | <img width="400" alt="Screenshot 2023-05-12 at 3 53 30 PM" src="https://github.com/Uniswap/interface/assets/89173284/ca91c22c-34f5-4344-b154-8fe4bb4fcd92">  | <img width="400" alt="Screenshot 2023-05-12 at 3 53 30 PM" src="https://github.com/Uniswap/interface/assets/89173284/716752de-2e3d-4d3f-a27b-3c0d0dbff5de"> |


## Testing
1. Go to app.uniseap.org on safari
2. Click on share site
3. Click on "Add to Home Screen
4. Open App

#### Automated testing
- [x] Unit test N/A
- [x] Integration/E2E test N/A

JFrankfurt commented on src/theme/index.tsx:
```diff
@@ -85,7 +85,7 @@ function getSettings(darkMode: boolean) {
 
     navHeight: 72,
-    mobileBottomBarHeight: 52,
+    mobileBottomBarHeight: 70,
```
```suggestion
    mobileBottomBarHeight: 68,
```

`72` would also work here

It's not documented well, but we try to keep all px values to increments of `4`

infredible replied:
Jordan's right about us trying to use a 4px base. But this should be even taller @ `76` to have more spacing between the tab labels and the home screen bar.

infredible commented:
Thanks for catching this @Jdecristi . Other than my comment above, your fix looks good when the app is added to the home screen, but we don't want this height when the app is viewed directly in mobile safari. 

This is what my chat with GPT said:

> Yes, you can change the height of an element depending on whether it is being viewed in the Mobile Safari app or a PWA. Here's how you can achieve this using JavaScript:
> 
> Detecting the PWA environment: In your JavaScript code, you can check if the current environment is a PWA by checking the window.matchMedia function. PWAs usually have a specific media query that can be used for detection. Here's an example:
> ```
> const isPWA = window.matchMedia('(display-mode: standalone)').matches;
> ```
> The isPWA variable will be true if the current environment is a PWA and false otherwise.
> 
> Changing the height of the element: Once you have determined the environment, you can use JavaScript to change the height of the element accordingly. Here's an example:
> 
> ```
> const element = document.getElementById('your-element-id');
> if (isPWA) {
>   // Set height for PWA
>   element.style.height = '300px';
> } else {
>   // Set height for Mobile Safari
>   element.style.height = '200px';
> }
> ```
> In this example, your-element-id should be replaced with the actual ID of the element you want to modify. The if-else statement sets different heights for the element based on whether it is viewed in a PWA or Mobile Safari.
> 
> By combining these steps, you can conditionally change the height of an element depending on whether it is being viewed in the Mobile Safari app or a PWA.

Jdecristi commented:
@infredible @JFrankfurt I fixed the issues you brought up. I changed the height from 70px to 76px and added a ternary to check if the app is PWA or not

kristiehuang commented:
hey - looks like this got stamped but didn't get merged! Closing due to age but feel free to resolve merge conflicts & reopen.
